config/features/config.go

```diff
@@ -254,6 +256,12 @@ func ConfigureBeaconChain(ctx *cli.Context) error {
 		logEnabled(BlobSaveFsync)
 		cfg.BlobSaveFsync = true
 	}
+	// For the p.o.c we enable it by default.
+	cfg.EnablePeerDAS = true
```

Comment:
Might want to be careful with this one. Don't merge everything to develop later thinking it's feature-flag gated but forget this line still exists.

Reply:
Oh yeah definitely, in any case even if we merge to develop a node with this enabled will immediately stall since all the topics are non-existent in mainnet or any of the testnets